karabo_data/geometry2.py

```diff
@@ -1050,6 +1138,22 @@ def to_distortion_array(self):
         """
         return super().to_distortion_array()  # Overridden only for docstring
 
+    @classmethod
+    def _tile_dims(cls, moduleno, tileno):
+        # Which part of the array is this tile?
+        module_offset = moduleno * 256
+        if tileno < 8: # First half of module (0 <= t <=7)
+            fs_dims = 0, 127
+            tiles_up = 7 - tileno
+        else:
+            fs_dims = 128, 255
+            tiles_up = tileno - 8
+
+        tile_offset = module_offset + (tiles_up * 32)
```

Comment:
I don't think you need to use `module_offset` for this. The virtual datasets we generate have a separate array dimension for the modules (panels in CrystFEL terminology). So the slow-scan/fast-scan coordinates we record are only within each module (0 to 255 in both dimensions for LPD).

Reply:
Ok!